test/OpenTelemetry.Tests/Metrics/MetricExporterTests.cs

```diff
@@ -65,6 +69,55 @@ public void FlushMetricExporterTest(ExportModes mode)
             }
         }
 
+        [Fact]
+        public void ExporterShouldNotUpdateMetricPoint()
+        {
+            var exportedItems = new List<Metric>();
+            var updateMetricPointExporter = new UpdateMetricPointExporter(exportedItems);
+            using var meter = new Meter(Utils.GetCurrentMethodName());
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddMeter(meter.Name)
+            .AddReader(new BaseExportingMetricReader(updateMetricPointExporter) { Temporality = AggregationTemporality.Delta })
+            .Build();
+
+            var counter = meter.CreateCounter<long>("counter");
+            counter.Add(100, new KeyValuePair<string, object>("key", "value"));
+
+            meterProvider.ForceFlush();
+
+            Assert.Single(exportedItems);
+            var metricPoint = this.GetFirstMetricPointFromMetric(exportedItems[0]);
+            Assert.Equal(100, metricPoint.GetSumLong());
+            foreach (var tag in metricPoint.Tags)
+            {
+                Assert.Equal("key", tag.Key);
+                Assert.Equal("value", tag.Value);
+            }
+
+            exportedItems.Clear();
+
+            counter.Add(150, new KeyValuePair<string, object>("key", "value"));
+
+            meterProvider.ForceFlush();
+
+            Assert.Single(exportedItems);
```

Comment:
At this point, `ForceFlush()` has called the `Export` method of the exporter and the exporter has updated the original `MetricPoint` to `default`.